ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -135,7 +135,15 @@ tests:
   container:
     from: src
 - as: e2e-aws
-  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  commands: |
+    fips_check
+    TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+- as: e2e-aws-fips
+  commands: |
+    fips_check
+    TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
@wking here and https://github.com/openshift/release/blob/3a81254561f1538c2c4945368bf20cdfa7d1f713/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L222-#L238